baybe/searchspace/discrete.py

```diff
@@ -698,11 +623,7 @@ def get_candidates(
             representation.
         """
         # Filter the search space down to the candidates
-        mask_todrop = self.metadata["dont_recommend"].copy()
-        if not allow_repeated_recommendations:
-            mask_todrop |= self.metadata["was_recommended"]
-        if not allow_recommending_already_measured:
-            mask_todrop |= self.metadata["was_measured"]
+        mask_todrop = self._excluded.copy()
```

Comment:
just to make sure:
this function will not take care anymore about anything regarding already recommended or measured points, right?
That functionality has not just been deleted but shifted to somewhere else (cant quickly find it in the dreadful review interface) ?
the flags are still there because they will be shifted in another PR?